NLibrary/Classes/PlaylistMVVM/PlaylistModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistBasicModel {
+//    public var playlistItems: [PlaylistModel] = []
+//    public var authorId: String?
+//    init?(dict: NSDictionary) {
+//        guard let keyValues = dict.allValues.first as? NSDictionary,
+//            let authorId = dict.allKeys.first as? String else {
+//            return
+//        }
+//        self.authorId = authorId
+//        for (key, value) in keyValues {
+//            guard let key = key as? String,
+//                let page = value as? NSDictionary else {
+//                return
+//            }
+//            playlistItems.append(PlaylistModel(albumName: key, dictionary: page))
+//        }
+//    }
+//    var dict: [String: Any]?
+}
+
+public struct PlaylistModel {
+    public var playlistId: String = ""
+    public var playlistName: String = ""
+    public var producerEmail: String = ""
+    public var producerId: String = ""
+    public var numSongs: Int = 0
+    public var basicSongItems: [BasicSongItem] = []
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5415919)